pages/common/guacd.md

```diff
@@ -0,0 +1,21 @@
+# guacd
+
+> Apache Guacamole proxy daemon.
+> Support loader for client plugins to interface between the Guacamole protocol and any arbitrary remote desktop protocol (e.g. RDP, VNC, Other).
+> More information: <https://guacamole.apache.org/>.
+
+- Bind guacd at specific port on localhost:
```

Comment:
Perhaps this will read better?

```suggestion
- Bind to a specific port on localhost:
```

Also, are there any long-forms of the options here? In tldr-pages we prefer to use long options over the short variants for clarity. The same applies tot he examples further down too.

Reply:
Unfortunately no, but I can raise that issue with the Guacamole Project and come back to update this page should they wish to implement them! (I'll even see if I can add it for them...)